AK/StringView.h

```diff
@@ -132,6 +132,15 @@ class StringView {
 
     [[nodiscard]] Vector<StringView> split_view_if(Function<bool(char)> const& predicate, bool keep_empty = false) const;
 
+    // FIXME: Implement find_last_split_view(StringView) for symmetry
+    [[nodiscard]] Optional<StringView> find_last_split_view(char separator) const
+    {
+        auto begin = find_last(separator);
+        if (!begin.has_value())
+            return {};
+        return substring_view(begin.release_value());
```

Comment:
This needs to be `begin.release_value() + 1` (or whatever the equivalent of "skip one" is), doesn't it?

At least the CI failure looks like it is caused by that, because all process names have a leading `/`.

Reply:
Yeah sure, that makes sense